src/packages/cli/src/Init.ts

```diff
@@ -73,7 +74,6 @@ export class Init implements Command {
     }
 
     const outputDir = process.cwd()
-    const prismaFolder = path.join(outputDir, 'prisma')
```

Comment:
Why don't you check for the prisma dir anymore?

Reply:
Didn't think it was necessary as we are moving away from that convention. I'll re-add the check if you think it is necessary

Reply:
That is still our convention, but users can customize.
We'll generate into `prisma/schema.prisma`, so we need to check if the file is already there.